_posts/2016-xx-xx-release-0.16.md

```diff
@@ -119,6 +119,116 @@ If you are using the low-level qcprot algorithm your self intead of our provided
 wrappers you have to change your code since the API has changed. For more see
 the [CHANGELOG].
 
+## MemoryReader: Reading trajectories from memory
+
+MDAnalysis typically reads trajectories from files on-demand, so that it can efficiently deal with large trajectories - even those that do not fit in memory. However, in some cases, both for convenience and for efficiency, it can be an advantage to work with trajectories directly in memory. In this release, we have introduced a MemoryReader, which makes this possible.
+
+The MemoryReader works with numpy arrays, using the same format as that used by for instance `DCDReader.timeseries()`. You can create a Universe directly from such an array:
```

Comment:
Just a comment/question: `.timeseries()` is currently only implemented for DCDs; there's an open issue for Gromacs formats MDAnalysis/mdanalysis#186. 

I think, it would make the post too complicated to mention here that this particular example only works with DCDs but for the future we need to think if we want to support `.timeseries()` more widely. The `Universe(..., in_memory=True)` approach takes care of this, of course.